lib/maze/schemas/trace_validator.rb

```diff
@@ -5,6 +5,10 @@
 module Maze
   module Schemas
 
+    HEX_STRING_16 = '^[A-Fa-f0-9]{16}$'
+    HEX_STRING_32 = '^[A-Fa-f0-9]{32}$'
+    SAMPLING_HEADER = '^(1(.0)?|0(\.[0-9]+)?):[0-9]+$'
```

Comment:
If there's more than 1 sampling probability then they are all listed in the header separated by semicolons, i.e. `a:b;c:d`, `a:b;c:d;e:f` etc&hellip;

Reply:
Regex updated and new test cases added.